K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -138,6 +143,53 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
             }
         }
 
+        /// <summary>
+        /// Get extended stats for standard deviation aggregate from a given <see cref="DataRow"/>.
+        /// </summary>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="row">The row to be parsed.</param>
+        /// <param name="logger">ILogger object for logging.</param>
+        /// <returns><see cref="ExtendedStatsAggregate"></returns>
+        private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnName, DataRow row, ILogger logger)
+        {
+            logger.LogTrace("Get extended stats for standard deviation aggregate for {}", columnName);
+
+            var extendedStatsAggregate = new ExtendedStatsAggregate();
+
+            var jObject = (JObject)row[columnName];
+
+            if (jObject == null)
+            {
+                return extendedStatsAggregate;
+            }
+
+            if (jObject.ContainsKey(AggregationsConstants.StandardDeviationBoundsUpper))
+            {
+                var upper = jObject[AggregationsConstants.StandardDeviationBoundsUpper].Value<double?>();
+                extendedStatsAggregate.StandardDeviationBounds.Upper = upper ?? (double.IsNaN(upper.Value) ? null : upper);
```

Comment:
Why the special nan handling? I tried giving it NaN and it doesn't return an error or anything. You can remove this check

Reply:
If we want to have the exact same payload from elastic and from adx, we must handle Nan values.
I mean, when Elastic is returning null, ADX is returning NaN.
So far, handling the NaN value ensures we have the same payload

Reply:
Ok, understood.